scripts/metavar.py

```diff
@@ -1677,12 +1677,37 @@ def add_variable(self, newvar, run_env, exists_ok=False, gen_unique=False,
                                        context=newvar.context)
             # end if
         # end if
+        # Check if local_name exists in Group. If applicable, Create new 
+        # variable with uniquie name. There are two instances when new names are
```

Comment:
```suggestion
        # Check if local_name exists in Group. If applicable, create new 
        # variable with unique name. There are two instances when new names are
```

Reply:
@dustinswales - can you address these typos?